DataFormats/TrackerRecHit2D/interface/FastTrackerRecHit.h

```diff
@@ -49,7 +49,7 @@ class FastTrackerRecHit : public BaseTrackerRecHit
     
     /// destructor
     ///
-    ~FastTrackerRecHit() {}
+    virtual ~FastTrackerRecHit() {}
```

Comment:
Still true, but the base class `TrackingRecHit` already has a virtual destructor (https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/DataFormats/TrackingRecHit/interface/TrackingRecHit.h#L67). Essentially the "virtualness" of a destructor is inherited.

(nevertheless some coding styles suggest to explicitly declare the destructors of derived classes virtual to make it more clear that the class hierarchy is polymorphic)